config/routes.rb

```diff
@@ -1,7 +1,10 @@
 Rails.application.routes.draw do
   devise_for :users
 
-  resources :items, only: [:index, :new, :create]
+  #itemsコントローラについてのルーティング
+  resources :items, only: :index
```

Comment:
```suggestion
  resources :items, only: [:index, :new, :create]
```
こちらレビュワーの指摘が一部誤っておりました。
現在、商品出品機能が終わり、商品一覧機能を実装しているため、
`new, createアクション`は記述していて問題ございません。記述を戻していただけると幸いです。
お手数をおかけして申し訳ございません。